docs/topics/plugins/versioning.md

```diff
@@ -0,0 +1,315 @@
+[//]: # (title: Versioning plugin)
+
+Versioning plugin aims to provide the ability to host documentation for multiple versions of your library/application
+with seamless switching between them. This, in turn, provides better experience for your users.
+
+![Screenshot of documentation version dropdown](versioning-plugin-example.png){height=350}
+
+> Versioning plugin only works with [HTML](html.md) format.
+> 
+{type="note"}
+
+## Applying the plugin
+
+You can apply Versioning plugin the same way as other [Dokka plugins](plugins_introduction.md#applying-dokka-plugins):
```

Comment:
Can you put a notice that in multimodule projects versioning plugin should be applied in all submodules?

Reply:
Indeed, fixed